requirements.txt

```diff
@@ -0,0 +1,4 @@
+loguru == 0.6.0
+transformers == 4.39.3
+torch ~= 2.2.0
+torch-xla[tpu] ~= 2.2.0 -f https://storage.googleapis.com/libtpu-releases/index.html
```

Comment:
FWIW, when the JAX supports gets into this library, libtpu alignment between JAX and torch_xla becomes important. JAX nighly supports the nightly libtpu. TorchXLA, currently, uses a "recent" libtpu pin. The pin gets updated each time we update the openxla pin. OpenXLA pin updates happen several times / month.